src/fides/api/models/privacy_preference.py

```diff
@@ -812,23 +883,43 @@ class LastServedNotice(LastSavedMixin, Base):
         ),
         UniqueConstraint(
             "provided_identity_id",
-            "vendor",
-            name="last_served_identity_vendor",
+            "vendor_consent",
+            name="last_served_identity_vendor_consent",
+        ),
+        UniqueConstraint(
+            "provided_identity_id",
+            "vendor_legitimate_interests",
+            name="last_served_identity_vendor_leg_interests",
+        ),
+        UniqueConstraint(
+            "fides_user_device_provided_identity_id",
+            "vendor_consent",
+            name="last_served_fides_user_device_identity_vendor_consent",
         ),
         UniqueConstraint(
             "fides_user_device_provided_identity_id",
-            "vendor",
-            name="last_served_fides_user_device_identity_vendor",
+            "vendor_legitimate_interests",
+            name="last_served_fides_user_device_identity_vendor_leg_interests",
         ),
         UniqueConstraint(
             "provided_identity_id",
-            "system",
-            name="last_served_identity_system",
+            "system_consent",
+            name="last_served_identity_system_consent",
+        ),
+        UniqueConstraint(
+            "provided_identity_id",
+            "system_legitimate_interests",
+            name="last_served_identity_system_leg_interests",
+        ),
+        UniqueConstraint(
+            "fides_user_device_provided_identity_id",
+            "system_consent",
+            name="last_served_fides_user_device_identity_system_consent",
         ),
         UniqueConstraint(
             "fides_user_device_provided_identity_id",
-            "system",
-            name="last_served_fides_user_device_identity_system",
+            "system_legitimate_interests",
+            name="last_served_fides_user_device_identity_system_leg_interests",
```

Comment:
okay I have combed through this file and didn't find any copy/paste errors, nice job!